nautobot_ssot/integrations/device42/diffsync/adapters/device42.py

```diff
@@ -677,7 +682,7 @@ def load_ip_addresses(self):
         self.job.logger.info("Loading IP Addresses from Device42.")
         _cfs = self.device42.get_ipaddr_custom_fields()
         for _ip in self.device42.get_ip_addrs():
-            _ipaddr = f"{_ip['ip_address']}/{str(_ip['netmask'])}"
+            _ipaddr = f"{_ip['ip_address']}/{_ip['netmask']}"
```

Comment:
How confident are you in this change?

Reply:
I'm fairly confident in this. I honestly can't recall why I would cast it to string when using f-strings should automatically do that.